src/setup_node_env/root/index.js

```diff
@@ -17,12 +17,5 @@
  * under the License.
  */
 
```

Comment:
Instead of changing this file, we could also remove line 20 of `src/setup_node_env/index.js` to have the same effect. That way, we could leave this code in, it just wouldn't ever be called.

I'm not at all opposed to leaving this as-is though!

Reply:
I like that better.

Reply:
Done